### DISCUSSION TITLE
utils/serve_logs.py flask app only for task logs?

### DISCUSSION THREAD

**Discussion by notacoat** [apache/airflow#23936] [Q&A]
Hello, 

Is the tiny flask app in `utils/serve_logs.py` only used to display task logs in the UI?  The reason I ask is that my team is using a custom logger that overrides the `base_log_folder`.  As a result, we cannot view logs in the UI (we are using celery) if the log is not on the specific host we are accessing the webserver.  

```
LOGGING_CONFIG["handlers"].update(
    {
        "task": {
            "class": "airflow.utils.log.file_task_handler.FileTaskHandler",
            "formatter": "airflow",
            "base_log_folder": f"{BASE_LOG_FOLDER}/dags",
            "filename_template": FILENAME_TEMPLATE,
            "filters": ["mask_secrets"],
        },
    }
)```

**Comment by potiuk** [comment]
Yes. It's only used for that.
